extra_data/reader.py

```diff
@@ -1642,7 +1644,26 @@ def train_timestamps(self, labelled=False):
         arr[arr == epoch] = 'NaT'  # Not a Time
         if labelled:
             import pandas as pd
-            return pd.Series(arr, index=self.train_ids)
+            series = pd.Series(arr, index=self.train_ids).dt.tz_localize('UTC')
+            return series.dt.tz_convert('Europe/Berlin') if euxfel_local_time else series
+        elif pydatetime:
+            from datetime import datetime, timezone
+            res = []
+            for npdt in arr:
+                pydt = npdt.astype('datetime64[ms]').item()
+                if pydt is not None:  # Numpy NaT becomes None
+                    pydt = pydt.replace(tzinfo=timezone.utc)
+                    if euxfel_local_time:
+                        from zoneinfo import ZoneInfo
```

Comment:
Oops, looks like I missed that `zoneinfo` was only added in 3.9. The test for 3.8 fails expectedly at this point. Shall we use `pytz` instead, which is likely already installed as a dependency of something?

Reply:
I think we could also drop 3.8 support - IIRC we were keeping it for calibration, but calibration has moved to Python 3.11. @philsmt do you know of any other reason we need to keep 3.8?

Reply:
Dropping 3.8 and sticking with the standard library would be my preference if that is possible.